migrator/migrations/policymigrationhelper/policy_migrator.go

```diff
@@ -246,6 +256,13 @@ func diffPolicies(beforePolicy, afterPolicy *storage.Policy) (PolicyUpdates, err
 	beforePolicy.Remediation = ""
 	afterPolicy.Remediation = ""
 
+	// Severity
+	if beforePolicy.GetSeverity() != afterPolicy.GetSeverity() {
+		updates.Severity = afterPolicy.Severity
+	}
+	beforePolicy.Severity = 0
+	afterPolicy.Severity = 0
```

Comment:
Couple of questions about the migration code:
1. Why don't we also have a check to migrate the new `Fixable` criterion?
2. Should we also migrate to `disabled: true`, or is that too risky, because existing customer may be relying on it being on?

Reply:
Both will be migrated. The Fixable check is part of the policy sections migration. And disabled: true is migrated as well. Line 267 in policy migrator.